src/promptflow/promptflow/executor/flow_executor.py

```diff
@@ -951,11 +967,17 @@ def _apply_inputs_mapping_for_all_lines(
         }]
         """
         if inputs_mapping is None:
-            # This exception should not happen since we should use default inputs_mapping if not provided.
-            raise NoneInputsMappingIsNotSupported("Inputs mapping is None.")
+            # This exception should not happen since we will use default inputs_mapping for None input.
+            raise NoneInputsMappingIsNotSupported(
+                message_format="Inputs mapping is None. "
+                "Please set one inputs mapping or use default inputs mapping in flow_executor."
+            )
         merged_list = FlowExecutor._merge_input_dicts_by_line(input_dict)
         if len(merged_list) == 0:
-            raise EmptyInputAfterMapping("Input data does not contain a complete line. Please check your input.")
+            raise InputMappingError(
+                message_format="The input for flow is incorrect. All inputs come from different line of data "
+                "so can't execute. Please provide data in same line to correct the input."
```

Comment:
I think this error message can be refined to be more native. We may ask for ChatGPT's help.

Reply:
@chw-microsoft 
Do we have template for system error's message about asking support for help?
Below is the most similar message for me.
![image](https://github.com/microsoft/promptflow/assets/17527303/dc5c3167-6176-4983-a956-361d16efe652)